### PR TITLE
framework/messaging : Change global variable to static

### PR DESCRIPTION
**Pull request by jeongchanKim**
Change g_port_info_list to static and add get function for using in another file.
port_list_sem is already used in only that file. just change to static.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [0fec94388b3491079401b0dcf02474c40f05604a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0fec94388b3491079401b0dcf02474c40f05604a] - Code Rule Check OK.

File: framework/src/messaging/messaging_internal.h [hunk 1]
```diff
@@ -137,6 +137,10 @@ void messaging_run_callback(int signo, siginfo_t *data);
  * @brief Internal function for parsing received packet
  */
 int messaging_parse_packet(char *packet, char *buf, int buflen, pid_t *sender_pid, int *msg_type);
+/**
+ * @brief Internal function for getting g_port_info_list
+ */
+sq_queue_t *messaging_get_port_info_list(void);
```
Comment (sunghan-chang) [inline framework/src/messaging/messaging_internal.h]: need to include <queue.h>

**Comment by seinfra** [comment]
Target : [a3a736c80597d191d8ae2f814ef90a7184fc5a45] - Code Rule Check (C++) OK.

**Pull request merged**